.travis.yml

```diff
@@ -0,0 +1,98 @@
+sudo: false
+
+language: cpp
+
+addons:
+  apt:
+    sources: &toolchains
+      - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty-5.0
+
+matrix:
```

Comment:
How come only C++14 is being tested?

Reply:
i think c++14 is default theses days. and because of x3.
but now i think x3 is not tested, because it's not part of test/Jamfile.
we can add much more entries to the matrix...

Reply:
I work on projects ranging from C++03 to C++14.  We still have requests to support Visual Studio 2008.  I wouldn't assume C++14 is the default for anybody.

Reply:
I think we need to add a lot more compiler versions and c++ dialects
- after build system integration works fine
- after separating tests into smaller parts to run more matrix builds (hopefully parallel)

For now I am very happy to see ANY results...